src/test/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheckTest.java

```diff
@@ -212,7 +212,7 @@ public void testLogIoExceptionFileNotFound() throws Exception {
         final Method loadKeys =
             check.getClass().getDeclaredMethod("getTranslationKeys", File.class);
         loadKeys.setAccessible(true);
-        final Set<String> keys = (Set<String>) loadKeys.invoke(check, new File(""));
+        final Set<String> keys = (Set<String>) loadKeys.invoke(check, new File(".no.such.file"));
```

Comment:
Why did this change?

Reply:
The new code is
```
inStream = Files.newInputStream(file.toPath());
```
If the file name is empty, the exception will be "Not a file". The old code throws "File not found", which is explicitly handled by `logIoException`.
With a valid (but not existent) file name the exception is "File not found".